### PR TITLE
fix(content): [Angular] Change the id_token result title

### PR DESCRIPTION
**Pull request by yasinmiran**
## Changes
- Alter the `id_token` content section title. This PR aims to add changes made in https://github.com/asgardeo/asgardeo-auth-react-sdk/pull/20 to Angular samples.

### CODE REVIEW

File: samples/asgardeo-angular-app/src/app/app.component.html [hunk 1]
```diff
@@ -76,7 +76,15 @@ <h3>
                 </ng-template>
 
                 <ng-template #defined>
-                    <h2>Authentication response derived by the Asgardeo Auth Angular SDK</h2>
+                  <h2>Authentication Response</h2>
+                  <h4 class="sub-title">
+                    Derived by the&nbsp;
+                    <code class="inline-code-block">
+                      <a href="https://www.npmjs.com/package/@asgardeo/auth-react/v/latest" target="_blank">
```
Comment (JanithGan) [inline samples/asgardeo-angular-app/src/app/app.component.html:83]:
```suggestion
                      <a href="https://www.npmjs.com/package/@asgardeo/auth-angular/v/latest" target="_blank">
```

File: samples/asgardeo-angular-app/src/app/app.component.html [hunk 2]
```diff
@@ -76,7 +76,15 @@ <h3>
...
+                        @asgardeo/auth-react
```
Comment (JanithGan) [inline samples/asgardeo-angular-app/src/app/app.component.html:84]:
```suggestion
                        @asgardeo/auth-angular
```

File: samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.html [hunk 3]
```diff
@@ -25,7 +25,15 @@
     </ng-template>
 
     <ng-template #defined>
-        <h2>Authentication response derived by the Asgardeo Auth Angular SDK</h2>
+        <h2>Authentication Response</h2>
+        <h4 class="sub-title">
+            Derived by the&nbsp;
+            <code class="inline-code-block">
+                <a href="https://www.npmjs.com/package/@asgardeo/auth-react/v/latest" target="_blank">
```
Comment (JanithGan) [inline samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.html:32]:
```suggestion
                <a href="https://www.npmjs.com/package/@asgardeo/auth-angular/v/latest" target="_blank">
```

File: samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.html [hunk 4]
```diff
@@ -25,7 +25,15 @@
...
+                    @asgardeo/auth-react
```
Comment (JanithGan) [inline samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.html:33]:
```suggestion
                    @asgardeo/auth-angular
```

**Review by JanithGan — CHANGES_REQUESTED** [review]

File: samples/asgardeo-angular-app/src/app/app.component.html (hunk 1 above)
Comment (yasinmiran) [reply samples/asgardeo-angular-app/src/app/app.component.html]: Resolved this content error in commit 86cd42f.

File: samples/asgardeo-angular-app/src/app/app.component.html (hunk 2 above)
Comment (yasinmiran) [reply samples/asgardeo-angular-app/src/app/app.component.html]: Resolved this content error in commit 86cd42f.

File: samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.html (hunk 3 above)
Comment (yasinmiran) [reply samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.html]: Resolved this content error in commit 86cd42f.

File: samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.html (hunk 4 above)
Comment (yasinmiran) [reply samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.html]: Resolved this content error in commit 86cd42f.

**Review by JanithGan — APPROVED** [review]

**Review by thivi — APPROVED** [review]

**Pull request merged**